express/scripts/templates.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+async function fetchPageContent(path) {
+  if (!(window.templates && window.templates.data)) {
+    window.templates = {};
+    const resp = await fetch('/express/templates/template-pages.json');
+    window.templates.data = resp.ok ? (await resp.json()).data : [];
+  }
+
+  return window.templates.data.find((p) => p.path === path && p.live === 'Y');
```

Comment:
As a safety precaution, I'd turn it around:
```suggestion
  return window.templates.data.find((p) => p.path === path && p.live !== 'N');
```